docs/general.md

```diff
@@ -24,7 +24,7 @@ By default, asynchronous continuations and retries do not execute on a captured
 ResilienceContext context = ResilienceContextPool.Shared.Get(continueOnCapturedContext: true);
 
 await pipeline.ExecuteAsync(
-    async context =>
+    static async context =>
```

Comment:
you can have static lambdas? hmmm

Reply:
Yup, when you want to be sure to never allocate on hot-path